src/components/VideoCard/VideoCard.jsx

```diff
@@ -0,0 +1,95 @@
+import "./VideoCard.css";
+import { Link } from "react-router-dom";
+import { useState } from "react";
+import { useDataValues } from "../../contextAndReducers/DataProvider";
+import { useWatchLaterActions } from "../../utilities/CustomHooks";
+function MenuCard({ obj }) {
+  const { makeWatchLater, deleteFromWatchLater } = useWatchLaterActions();
+  const { state } = useDataValues();
+  const { _id: id } = obj;
+  const findItemIndex = state.watchLater.findIndex((item) => item._id === id);
+  return (
+    <div className="menu-video-card">
+      <p className="menu-card-options">
+        <i className="material-icons md-dark padding-l-r">list</i>Add to
+        playlist
+      </p>
+      {findItemIndex === -1 && (
+        <p className="menu-card-options" onClick={() => makeWatchLater(obj)}>
+          <i className="material-icons md-dark md-18 padding-l-r">schedule</i>
+          Watch Later
+        </p>
+      )}
+      {findItemIndex > -1 && (
+        <p
+          className="menu-card-options"
+          onClick={() => deleteFromWatchLater(id)}
+        >
```

Comment:
I have found another way of checking whether an item exists or not. I remember telling you to use findIndex, but another way is to use the .some() method. The code will look as follows:
``` state.watchLater.some((item) => item._id === id) ```  This returns a boolean value
This is just an another method of doing things, thought I'd share this.

Reply:
Yeah I knew it earlier also Thanks for suggestion though